docs/index.md

```diff
@@ -11,18 +11,103 @@ on the files.
 <p>
 <b>Step 2: Accessing sequencing files for CRISPResso2</b><br>Once the setup notebook has been run the sequencing files 
 should be available at <b><a href='https://drive.google.com/drive/folders/1uMXOLjvfY9TNlhwj0fVcTp6k2-heQe0c'>"GPP Cloud 
->R&D>Sequencing_Backup"</a></b>. They can be downloaded from here on to your local machine for running CRISPResso2. 
-The FASTQC outputs can be found here <b><a href='https://drive.google.com/drive/folders/1NhOYq3_P2Jr3aj_K0KbU1f-9iSWBbKso'>
+>R&D>Sequencing_Backup"</a></b>. They can be downloaded from here on to your local machine for running CRISPResso2. The
+input files should all be in the folder where you will run CRISPResso2 from. It is recommended to create a new folder 
+for every new run. The FASTQC outputs can be found here <b><a href='https://drive.google.com/drive/folders/1NhOYq3_P2Jr3aj_K0KbU1f-9iSWBbKso'>
 "GPP Cloud >R&D>FASTQC_outputs"</a></b>.
 </p>
 
 <p>
-<b>Step 3: Setting up a batch file for CRISPResso2</b><br>Batch file for a CRISPResso2 run is a .txt file with various 
-columns indicating different parameters for each sample in the run. The different columns are as described below: 
-<ol>
-    <li>Generate a name of the format, "BEV_sample_number_primerpair" 
-for each sample. For example, the name for sample 1 would be BEV_001_F1_A1_R1_A1, where 001 is the sample number and 
-F1_A1 is the forward primer and R1_A1 is the reverse primer.
+<b>Step 3: Installing Docker to run CRISPResso2</b><br>Installing CRISPResso via Docker is the easiest way to use
+it. The way to do this is explained <a href="https://github.com/pinellolab/CRISPResso2#docker" target="_blank">here</a>.
+</p>
 
+<p>
+<b>Step 4: Setting up a batch file for CRISPResso2</b><br>Batch file for a CRISPResso2 run is a .txt file with various 
+columns indicating different parameters for each sample in the run. This batch file should also be placed in the folder 
+created for the CRISPResso2 run. The different columns in a batch file are as described below: 
+<ol>
+    <li><b>name</b>: Generate a name of the format, <font color="#8b0000"> BEV_sample_number_primerpair</font>
```

Comment:
super small: in the file name format maybe take out the '_' between "sample" and "number so it's clear that that's one thing? So it would be "BEV_samplenumber_primerpair"